prompttools/utils/autoeval_from_expected.py

```diff
@@ -34,6 +35,7 @@ def _get_messages(prompt: str, expected: str, response: str):
     ]
 
 
+# TODO: Should this be removed since no one is using it?
```

Comment:
Yeah I think we can remove this

Reply:
Someone is working on a closely related PR - but it shouldn't be too disruptive

https://github.com/hegelai/prompttools/pull/59